subsys/net/l2/ethernet/dsa/dsa.c

```diff
@@ -1,84 +1,24 @@
-/** @file
- * @brief DSA related functions
- */
-
 /*
- * Copyright (c) 2018 Intel Corporation
+ * Copyright 2025 NXP
```

Comment:
According to our discussion, let's add a "dsa_" prefix in order to follow the same naming rules in Zephyr driver, for example, use "dsa_core.c dsa_port.c dsa_salve.c ..."

Another suggestion, in order to make the old architectrue code and new architecuture code works simulteneously in new architecuture developing period,  let's don't modify the old file directly, we could create new architeccture code in each new clean file, so that we could define which files can be compiled in cmake according to CONFIG_NET_DSA_DEPRECATED.

Reply:
It makes sense. That will make code much clear. Will do that.
Thank you very much.